src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -56,6 +57,10 @@
     private final SecurityUtils securityUtils;
     private final RestTemplate restTemplate;
 
+    private static final String[] ROLE_WHITELIST = {
```

Comment:
If we want this to be more flexible we could take it from properties instead of hardcoded. (Not sure if was necessary for this ticket though?)

Reply:
Refactored to be in application Params 👍